src/kernel/arch/x86/interrupts.zig

```diff
@@ -49,11 +49,7 @@ export fn commonStub() callconv(.Naked) void {
     // If so don't change cr3 to avoid a TLB flush
```

Comment:
Is this comment still relavant, should the cr3 check need to be re-added.

Reply:
Yeah it should be re-added.